src/watchdog/src/watchdog.py

```diff
@@ -267,7 +267,7 @@ def collect_k8s_componentStaus(k8s_gauge, api_server_ip, api_server_port, nodesJ
         ip = name["metadata"]["name"]
 
         collect_healthz(k8s_gauge, kubelet_healthz_histogram,
-            "k8s_kubelet", ip, 10255, "/healthz")
+            "k8s_kubelet", ip, 10255, "/healthz", None, None)
```

Comment:
are you sure AKS kubelet doesn't requires authentication?

Reply:
Sure